src/scopes.ts

```diff
@@ -641,7 +641,7 @@ const SCOPES_WITHOUT_VIEW_ONLY: {
   },
   [ScopeName.ViewManagedConsentDatabaseAdminApi]: {
     title: 'View Managed Consent Database Admin Api',
-    dependencies: [ScopeName.ViewConsentManager],
+    dependencies: [],
```

Comment:
@linh-transcend i think you also want to make the viewConsentManager depends on this new scope? per mike's suggestion?

Reply:
my bad. just added!